### PR TITLE
Added SLE15 SP3 image descriptions

### PR DESCRIPTION
**Pull request by schaefi**
Development of SLE15 SP3 has started. This commit adds the
image descriptions to meet this target

### CODE REVIEW

**Review by rjschwei — COMMENTED** [review]
Comments are general and apply to both LI/VLI

File: images/li/sle15_sp3/config.kiwi [hunk 1]
```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- The line below is required in order to use the multibuild OBS features -->
+<!-- OBS-Profiles: @BUILD_FLAVOR@ -->
+<!-- OBS-ExclusiveArch: x86_64 -->
+<image schemaversion="6.2" name="SLES15-SP3-SAP-Azure-LI-BYOS" displayname="SLES15-SP3-SAP-Azure-LI-BYOS">
```
[the remark this answers is not in the document]
Comment (schaefi) [reply images/li/sle15_sp3/config.kiwi:6]: yes good point will auto convert them, thanks

File: images/li/sle15_sp3/config.kiwi [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+        <contact>public-cloud-dev@susecloud.net</contact>
+        <specification>SUSE Linux Enterprise Server 15 Service Pack 3 for SAP
+            Applications image for Microsoft Azure Large Instance - Bring Your Own Subscription</specification>
+    </description>
+    <profiles>
+        <profile name="Devel" description="Devel Image Build"/>
+        <profile name="Production" description="Production Image Build"/>
+    </profiles>
+    <preferences>
+        <version>1.0.10</version>
```
[the remark this answers is not in the document]
Comment (schaefi) [reply images/li/sle15_sp3/config.kiwi:18]: good point, yes will change it

File: images/li/sle15_sp3/root/etc/sysconfig/SuSEfirewall2 [hunk 3]
```diff
@@ -0,0 +1,1157 @@
+# Copyright (c) 2000-2002 SuSE GmbH Nuernberg, Germany
```
[the remark this answers is not in the document]
Comment (schaefi) [reply images/li/sle15_sp3/root/etc/sysconfig/SuSEfirewall2:1]: I think the resolver automatically does this, but I change the name. Thanks

[the remark this answers is not in the document]
Comment (schaefi) [reply images/li/sle15_sp3/root/etc/sysconfig/SuSEfirewall2:1]: done

File: images/li/sle15_sp3/config.sh [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+[ -x /sbin/set_polkit_default_privs ] && /sbin/set_polkit_default_privs
+
+# ssh settings are different than public Azure per Msft request
+# Set sshd keep alive interval
+sed -i 's/#ClientAliveInterval 0/ClientAliveInterval 180/' \
+    /etc/ssh/sshd_config
+
+# Remove the password for root
+# Note the string matches the password set in the config file
+sed -i 's/$1$wYJUgpM5$RXMMeASDc035eX.NbYWFl0/*/' \
```
[the remark this answers is not in the document]
Comment (schaefi) [reply images/li/sle15_sp3/config.sh]: done

File: images/li/sle15_sp3/config.kiwi (hunk 2 above)
[the remark this answers is not in the document]
Comment (schaefi) [reply images/li/sle15_sp3/config.kiwi]: done LI and VLI for SP3 starts at 0.9.0

File: images/li/sle15_sp3/config.kiwi (hunk 1 above)
[the remark this answers is not in the document]
Comment (schaefi) [reply images/li/sle15_sp3/config.kiwi]: done

**Comment by schaefi** [comment]
@rjschwei regarding firewalld. There is an overlay file etc/sysconfig/SuSEfirewall2 is that read if we use firewalld ?

**Comment by rjschwei** [comment]
> @rjschwei regarding firewalld. There is an overlay file etc/sysconfig/SuSEfirewall2 is that read if we use firewalld ?

No, the configuration from SuSEfirewall2 has to be translated to firewalld settings and we get a new overlay file.

**Comment by schaefi** [comment]
good to know. I'll try susefirewall2-to-firewalld tool

**Comment by schaefi** [comment]
@rjschwei I've done the conversion using the suse provided tooling. The result overlay files are done by a diff between the firewalld provided default config and the stuff produced by the conversion tool. If that is complete/functional I can't tell as it needs to run in the Microsoft network for testing. If you see anything directly missing let me know. The structure between the two services is pretty different and the conversion tool also has some exceptions which it does not convert

**Review by rjschwei — APPROVED** [review]
Let's run with it. We have plenty of time for testing and iteration.

**Pull request merged**